CONTRIBUTING.md

```diff
@@ -9,20 +9,32 @@ please checkout [`DESIGN.md`](DESIGN.md).
 1. Rust Nightly. [Install Rust]. Once Rust is installed, run
 
     ```
-    rustup default nightly
+    $ rustup default nightly
```

Comment:
so, fascinatingly enough- this usually trips people up because they think they can cut and paste the whole thing, which is why i omitted it. likely the people contributing to this project won't have that issue but it's sorta nice to standardize on.